btf_interp/src/lib.rs

```diff
@@ -1,13 +1,14 @@
 //! Provide interpreter implementation for BF program.
 use btf_types::BrainFuckProgram;
+use std::num::NonZeroUsize;
 
 /// Provide structure for Virtual Machine
 #[derive(Debug)]
 pub struct VirtualMachine<T> {
     /// The collection to store elements of the tape.
     tape: Vec<T>,
     /// The size of the tape.
-    tape_size: usize,
+    tape_size: NonZeroUsize,
```

Comment:
At this point, you can use `usize` which simplifies things later on.  Otherwise you keep having to `.get()` the value.